src/containers/Tenant/QueryEditor/QueryResult/QueryResult.js

```diff
@@ -94,31 +100,33 @@ function QueryResult(props) {
     };
 
     const renderIssues = () => {
-        const error = props.error?.data;
+        const error = props.error;
 
-        const hasIssues = error?.issues && Array.isArray(error.issues);
+        const hasIssues = error?.data?.issues && Array.isArray(error.data.issues);
 
         return hasIssues ? (
             <React.Fragment>
-                <ResultIssues data={error} />
+                <ResultIssues data={error.data} />
                 {isFullscreen && (
                     <Fullscreen>
                         <div className={b('result', {fullscreen: true})}>
-                            <ResultIssues data={error} />
+                            <ResultIssues data={error.data} />
                         </div>
                     </Fullscreen>
                 )}
             </React.Fragment>
         ) : (
-            <span>{error?.data ?? error}</span>
+            <div className={b('error')}>
+                {prepareQueryError(error)}
```

Comment:
It seems error could be undefined here. And `prepareQueryError` expects error to be defined. 

https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/utils/query.ts#L188

Reply:
fixed